creator-node/src/services/stateMachineManager/stateReconciliation/updateReplicaSet.jobProcessor.ts

```diff
@@ -581,4 +611,54 @@ const _isReconfigEnabled = (enabledReconfigModes: string[], mode: string) => {
   return enabledReconfigModes.includes(mode)
 }
 
+type ShouldReconfigParams = {
+  libs: any
+  oldPrimarySpId: number
+  oldSecondary1SpId: number
+  oldSecondary2SpId: number
+  userId: number
+  logger: any
+}
+const _shouldReconfig = async ({
```

Comment:
i'd rename this to _canReconfig because by virtue of being on this queue it should reconfig. this is to determine if it can if the state is consistent on chain